### PR TITLE
Update install-gui.sh to use check Node 18 and npm 9

### PR DESCRIPTION
**Pull request by paninaro**
### Purpose:
The GUI has been updated to use Node 18 and npm 9. This change updates the install-gui.sh to verify that Node 18 and npm 9 are installed.

### CODE REVIEW

**Review by hoffmang9 — DISMISSED** [review]
Lgtm

File: install-gui.sh [hunk 1]
```diff
@@ -67,17 +67,17 @@ do_install_npm_locally(){
     export N_PREFIX=${SCRIPT_DIR}/.n
     PATH="${N_PREFIX}/bin:$(npm bin):${PATH}"
```
Comment (wallentx) [inline install-gui.sh:68]: Is `npm bin` still valid in node18?

**Review by arvidn — DISMISSED** [review]

File: install-gui.sh (hunk 1 above)
Comment (wallentx) [reply install-gui.sh:68]: @paninaro I verified `npm bin` was removed in v9.0.0. It is suggested to use `npm exec foo` or `npx --yes foo`.

Comment (paninaro) [reply install-gui.sh:68]: @ChiaMineJP Can you take a look at the issue @wallentx mentioned? Feel free to make whatever changes you feel are necessary.

Comment (ChiaMineJP) [reply install-gui.sh]: I've replaced `$(npm bin)` with `$(npm prefix)/node_modules/.bin`.
Reference:
https://docs.npmjs.com/cli/v8/commands/npm-prefix
https://docs.npmjs.com/cli/v9/configuring-npm/folders?v=true

**Review by hoffmang9 — DISMISSED** [review]
lgtm

File: install-gui.sh (hunk 1 above)
Comment (wallentx) [reply install-gui.sh]: This actually may be proper, since the only way to reach this part of the code is if you have an `npm` version less than `v9.x.x`

**Review by wallentx — DISMISSED** [review]

**Comment by ChiaMineJP** [comment]
@hoffmang9 @wallentx 
Sorry everyone, I need to push one more commit to suppress this warning:
image

**Comment by ChiaMineJP** [comment]
Also needed to update this lines of code:
image

**Comment by ChiaMineJP** [comment]
Tested both `install-gui.sh` and `start-gui.sh` in Ubuntu 22.04 and confirmed it works.

**Review by hoffmang9 — DISMISSED** [review]
lgtm

**Comment by ChiaMineJP** [comment]
Test on very old NodeJS/npm failed with the newer format of `package-lock.json`. (Thank you William)
Execuse me, but one-last commit will be pushed.

**Review by wallentx — APPROVED** [review]
Tested on Arch:
- With a good version of node/npm installed ✅
- With an old version of node/npm installed ✅
- Without node/npm installed ✅

**Review by hoffmang9 — APPROVED** [review]

**Review by cmmarslender — APPROVED** [review]

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**